requirements/default.txt

```diff
@@ -1,2 +1,4 @@
+celery
+Django>=1.11.17
```

Comment:
You can't add them to default, or the tests will all fail.  The tests must install a particular version depending on what variables the test is running at the time, and pip complains if you mention the same package twice.  

What you'll probably want to do is update the setup.py file to add just "celery" or "celery<5.0.0" to the install requirements for python3 and python2 respectively.

Reply:
Alright, second try :smile: